<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AGK_QTClass</class>
 <widget class="QMainWindow" name="AGK_QTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AGK_QT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="title">
          <string>Map</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="3">
           <widget class="QPushButton" name="addCircle_pushButton">
            <property name="text">
             <string>Add circle</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="addBox_pushButton">
            <property name="text">
             <string>Add box</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="go_pushButton">
         <property name="text">
          <string>GO</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="clearAll_pushButton">
         <property name="text">
          <string>Clear all</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pause_pushButton">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>World</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Start dir</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="startPosX_doubleSpinBox"/>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="pickStartPos_pushButton">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Pick</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Start pos</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Population size</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QDoubleSpinBox" name="startDirY_doubleSpinBox"/>
          </item>
          <item row="1" column="4">
           <widget class="QDoubleSpinBox" name="startPosY_doubleSpinBox"/>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="startDirX_doubleSpinBox"/>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="popSize_spinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Neural Networks</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="saveNN_pushButton">
            <property name="text">
             <string>Save NN</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="loadNN_pushButton">
            <property name="text">
             <string>Load NN</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>8</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPopulationTableWidget" name="overview_tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_loadWorld"/>
    <addaction name="action_saveWorld"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_loadWorld">
   <property name="text">
    <string>Load World</string>
   </property>
  </action>
  <action name="action_saveWorld">
   <property name="text">
    <string>Save World</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QPopulationTableWidget</class>
   <extends>QTableWidget</extends>
   <header>QPopulationTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="AGK_QT.qrc"/>
 </resources>
 <connections/>
</ui>
